framework/src/executioners/Transient.C

```diff
@@ -736,7 +736,8 @@ Transient::setupTimeIntegrator()
     case 2: ti_str = "CrankNicolson"; break;
     case 3: ti_str = "BDF2"; break;
     case 4: ti_str = "ExplicitMidpoint"; break;
-    case 5: ti_str = "LStableDirk2"; break;
+    case 5: ti_str = "ExplicitTVDRK2"; break;
+    case 6: ti_str = "LStableDirk2"; break;
```

Comment:
And also line 52 in `Transient.C` is out of sync with this. It is missing the new `TVDRK2` scheme. I suggest to use `explicit-tvd-rk-2`. The dash between `rk` and `2` is to be consistent with our `rk-2` name.